core/src/main/java/org/apache/accumulo/fate/util/Retry.java

```diff
@@ -67,6 +67,16 @@ private Retry(long maxRetries, long startWait, long waitIncrement, long maxWait,
     this.logIntervalNanoSec = MILLISECONDS.toNanos(logInterval);
     this.hasNeverLogged = true;
     this.lastRetryLog = -1;
+    this.waitFactor = 1;
+  }
+
+  public long getWaitFactor() {
```

Comment:
I'm not sure we need a public getter for this, since this should be used internally only.